metalearners/utils.py

```diff
@@ -73,3 +79,29 @@ def simplify_output(tensor: np.ndarray) -> np.ndarray:
     if n_outputs == 2:
         return tensor[:, :, 1].reshape(n_obs, n_variants)
     return tensor
+
+
+class FixedBinaryPropensity(ClassifierMixin, BaseEstimator):
+    """Binary classifier propensity dummy model which outputs a fixed propensity,
+    independently of covariates."""
+
+    def __init__(self, propensity_score: float) -> None:
```

Comment:
My only nitpick is that this allows only for a single fixed propensity score. As you write [here](https://metalearners.readthedocs.io/en/latest/examples/example_propensity.html), it would be nice to extend this class to allow for varying but fixed propensity scores, depending either on the index of X or some features.

Reply:
E.g. in my settings we have a mapping {model_id : prop_score} for around 25 different models.

Reply:
Thanks for your feedback! Let's tackle this in a follow-up :)